docs/send-data/kubernetes/collecting-logs.md

```diff
@@ -504,6 +504,128 @@ sumologic:
             manual_prefix_regexp: (\\{\".*|\\[?\\d{4}-\\d{1,2}-\\d{1,2}.\\d{2}:\\d{2}:\\d{2}.*)
 ```
 
+## Sending data to multiple targets
+
+It is possible to send logs to multiple locations. This section describes the following cases:
+
+- Send logs simultaneously to Sumo Logic and other locations.
+- Send all logs to Sumo Logic and part of it to other locations.
+- Send logs selectively to multiple locations.
+
+### Send logs simultaneously to Sumo Logic and other locations
+
+In order to send data to Sumo Logic and other locations, `sumologic.logs.otelcol.extraExporters` should be used.
+
+With the following configuration, data will be sent to both Sumo Logic and AWS S3:
+
+```yaml
+sumologic:
+  logs:
+    otelcol:
+      extraExporters:
+        ## define `awss3/production` exporter
+        ## as no routing is defined, all data will go to this expoter in addition to the default one
+        awss3:
+          s3uploader:
+              region: 'eu-central-1'
+              s3_bucket: 'databucket'
+              s3_prefix: 'metric'
+              s3_partition: 'minute'
+metadata:
+  logs:
+    statefulset:
+      extraEnvVars:
+        # Set env variables for awss3 exporter
+        - name: AWS_ACCESS_KEY_ID
+          value: "< YOUR AWS ACCESS KEY >"
+        - name: AWS_SECRET_ACCESS_KEY
+          value: "< YOUR AWS SECRET ACCESS KEY >"
```

Comment:
I'm not sure we should encourage this practice. At least in EKS, you would handle this authentication by adding an annotation to the Pod, allowing it to assume a given role with the right permissions. We should absolutely tell the user they need to ensure the necessary credentials are present in the Pod, though.

Reply:
I agree, we documented something similar in the cloudwatch collector.

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/templates/logs/collector/otelcol-cloudwatch/serviceaccount.yaml#L12

https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/docs/fargate.md#authenticate-with-cloudwatch

Reply:
I changed additional exporter to `sumologic/production` for all examples, as the main goal of this section is to explain concept of more complex routing than how to send data to AWS S3